### PR TITLE
PBL-32311 Create PDC Apis

### PR DESCRIPTION
**Pull request by HBehrens**
Adds all APIs to load, manipulate, and render PDCs and PDC sequences (Pebble's vector binary format). Also, adds missing `GColorClear`.

- [x] Attach resource loading via proxy

### CODE REVIEW

**Comment by HBehrens** [comment]
@franc0is

File: src/symbols-generated.js [hunk 1]
```diff
@@ -702,6 +703,368 @@ Rocky.addGeneratedSymbols = function(obj) {
     }
   };
 
+  // GDrawCommandType
+  obj.GDrawCommandTypeInvalid = 0;
+  obj.GDrawCommandTypePath = 1;
+  obj.GDrawCommandTypeCircle = 2;
+  obj.GDrawCommandTypePrecisePath = 3;
+
+  // constructs a function that calls hides:
```
Comment (franc0is) [inline src/symbols-generated.js]: That sentence does not make sense. that calls hides?

File: src/symbols-generated.js [hunk 2]
```diff
@@ -702,6 +703,368 @@ Rocky.addGeneratedSymbols = function(obj) {
...
+      image.releaseCPointer(cPtr);
+    }
+  };
+
+  // GSize gdraw_command_image_get_bounds_size(GDrawCommandImage * image)
+  // GSize *emx_gdraw_command_image_get_bounds_size(GDrawCommandImage *image);
+  obj.gdraw_command_image_get_bounds_size = firstArgCapturedCWrap(
+    'emx_gdraw_command_image_get_bounds_size',
+    'number', ['number'],
+    function(f, imagePtr) {
```
Comment (franc0is) [inline src/symbols-generated.js]: more descriptive name than f please. wrappedFunc?

File: src/symbols-generated.js (hunk 1 above)
Comment (HBehrens) [inline src/symbols-generated.js]: :white_check_mark:  no idea what I did there

File: src/symbols-generated.js (hunk 2 above)
Comment (HBehrens) [inline src/symbols-generated.js]: :white_check_mark: changed it here and in similar places

**Comment by franc0is** [comment]
:+1:

**Pull request merged**